src/components/TodoList/todolist.scss

```diff
@@ -0,0 +1,29 @@
+.todo-list {
+  width: 500px;
+  background-color: hsl(237deg, 14%, 26%);
+  list-style: none;
+  margin: 0px;
+  padding: 0px;
+
+  &__grid{
+    max-width: 100%;

```

Comment:
Just curiosity, why is the need to define max-width as 100%? 🤔

Reply:
before the changes i made, it was necessary, because it was passing trough the list box width.